recipe/meta.yaml

```diff
@@ -18,34 +17,47 @@ requirements:
   host:
     - python
     - pip
-    - hatchling
     - wheel
   run:
     - python
-    - importlib-metadata  # [py<38]
     - jinja2
-    - jsonschema >=3.0
+    - jsonschema >=3.0,<4.17
     - numpy
     - pandas >=0.18
     - toolz
-    - typing-extensions >=4.0.1  # [py<311]
+    - entrypoints
 
 test:
   imports:
     - altair
+    - altair.examples
     - altair.expr
+    - altair.sphinxext
+    - altair.tests
     - altair.utils
+    - altair.vega
+    - altair.vega.v5
+    - altair.vega.v5.schema
     - altair.vegalite
+    - altair.vegalite.v3
+    - altair.vegalite.v3.schema
+    - altair.vegalite.v4
+    - altair.vegalite.v4.schema
   source_files:
     - altair
   requires:
     - pytest
-    - vega_datasets
+    - sphinx
+    - docutils
+    - vega_datasets >=0.8
+    - recommonmark
     - ipython
     - pip
   commands:
     - pip check
-    - python -m pytest --pyargs --doctest-modules altair
+    # Temporarily skipping tests that use openSSL on Windows.
```

Comment:
Can you link to https://github.com/AnacondaRecipes/openssl-feedstock/pull/21 please?